x/params/subspace/subspace.go

```diff
@@ -281,27 +223,27 @@ type ReadOnlySubspace struct {
 	s Subspace
 }
 
-// Exposes Get
+// Get delegates a read-only Get call to the Subspace.
 func (ros ReadOnlySubspace) Get(ctx sdk.Context, key []byte, ptr interface{}) {
 	ros.s.Get(ctx, key, ptr)
```

Comment:
this looks like a store accessor pattern, which I'm not a fan of particularly...

Reply:
🤷‍♂  -- this PR has already expanded beyond its scope by me removing subkey functionality/APIs. I doubt `ReadOnlySubspace` is even used. We can investigate and potentially remove it in another PR.

Reply:
Sounds good. Let's do a clean up after this PR has been merged.